ui/tabs.js

```diff
@@ -843,7 +843,7 @@ return $.widget( "ui.tabs", {
 						that._trigger( "load", event, eventData );
 					}, 1 );
 				})
-				.complete(function( jqXHR, status ) {
+				.always(function( jqXHR, status ) {
```

Comment:
The arguments that the `always` function accepts is not the same as `complete`.  `jqXHR` is only the first argument in the error case;  in the success case, it's the last.